pkg/apis/awsprovider/v1alpha1/types.go

```diff
@@ -340,6 +335,40 @@ func (i IngressRules) Difference(o IngressRules) (out IngressRules) {
 	return
 }
 
+// Equal returns true if two IngressRule are equal
+func (i *IngressRule) Equal(o *IngressRule) bool {
```

Comment:
This method should probably be named `Equals`, wdyt?

Reply:
Now, that I think of it, you're right. Let me change to `Equals`.